core/types/confidential.go

```diff
@@ -89,6 +91,97 @@ func (tx *ConfidentialComputeRecord) setSignatureValues(chainID, v, r, s *big.In
 	tx.ChainID, tx.V, tx.R, tx.S = chainID, v, r, s
 }
 
+type ConfidentialComputeRequest2 struct {
+	EIP712    *eip712.EIP712TypedData `json:"eip712"`
+	Signature []byte                  `json:"signature"`
+	Sender    common.Address          `json:"sender"`
```

Comment:
This is hacky thing because Geth expects all the signatures in the v, r, s form but in this case is just []byte.